src/biogeophys/SnowHydrologyMod.F90

```diff
@@ -1232,7 +1238,9 @@ subroutine UpdateState_TopLayerFluxes(bounds, num_snowc, filter_snowc, &
           call endrun("In UpdateState_TopLayerFluxes, h2osoi_ice has gone significantly negative")
        end if
 
-       if (h2osoi_liq(c,lev_top(c)) < 0._r8) then
+       !       if (h2osoi_liq(c,lev_top(c)) < 0._r8) then
+       ! in some case roundoff level negative values trigger this 
+       if (h2osoi_liq(c,lev_top(c)) < -1.e-12_r8) then
```

Comment:
the 0 was changed to -1e-12 b/c I found some cases where <0 was catching values of h2osoi_liq that were negative roundoff values of zero due to the removal of the constrained evaporation from the available moisture being roundoff level different values.